### PR TITLE
Automated Test - COMMERCE-5856 | master

### PR DESCRIPTION
**Pull request by RitaSilveira**

### CODE REVIEW

**Comment by RitaSilveira** [comment]
https://issues.liferay.com/browse/COMMERCE-5856; CC:@lorenzolapescara

**Comment by kristinonias** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase [hunk 1]
```diff
@@ -105,7 +105,32 @@ definition {
 	test AssertProductCardImageRedirectsToProductDetailsPage {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase:104]: Please remove `@ignore = "true"`.

File: modules/apps/commerce/commerce-product-test/src/testFunctional/paths/CommerceFrontStore.path [hunk 2]
```diff
@@ -33,6 +33,11 @@
 	<td>//div[@class='product-gallery']//img[contains(@src, '${key_FSProductImage}')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>IMAGE_CARD_U_JOINT</td>
+	<td>//div[contains(@style,'u-joint')]</td>
+	<td></td>
+</tr>
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/paths/CommerceFrontStore.path:40]: This locator is too specific, please rename it "PRODUCT_CARD_IMAGE" and try to identify and use a more reusable path(e.g. by using a variable).

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]

**Pull request closed without merging**